include/vctr/Containers/VctrBase.h

```diff
@@ -35,16 +35,16 @@ class VctrBase : private Config,
     //==============================================================================
     // Retrieving size related information.
     //==============================================================================
+
+    // clang-format off
+
     /** Returns the extent of this instance, optionally shrank by a certain amount.
 
         In case the instance specifies a dynamic extent, the return value will always be
         std::dynamic_extent, no matter if we specified an amount to shrink.
         In case the instance specifies a static extent, the return value will be extent - amountToShrink.
      */
-    static consteval size_t getExtent (size_t amountToShrink = 0)
-    {
-        return extent == std::dynamic_extent ? std::dynamic_extent : (extent - amountToShrink);
-    }
+    static consteval size_t getExtent (size_t amountToShrink = 0) {  return extent == std::dynamic_extent ? std::dynamic_extent : (extent - amountToShrink);  }
```

Comment:
A lot of the methods in this file were not clang-format safe. This is why I added the guards. This one I aligned to the style of the other methods that are single-line in majority.

Reply:
I see... this one is quite long for a single liner in my opinion. Maybe we should even go and move all non-single liners (or really all definitions) to the .tpp file?

Reply:
I don't know - my main concern was here that the file wasn't clang-format safe. This might be a long line, but seems tolerable - in contrast to many others that have _multiple_ statements within one line, in this fashion:

```
type function() { VCTR_ASSERT (someghing); return someghingElse; }
```

Reply:
No leave it as it is, I'd rather add a general cleanup task for this file to the backlog

Reply:
Now it's too late :)